packages/typespec-test/test/loadTest/spec/client.tsp

```diff
@@ -0,0 +1,49 @@
+import "./main.tsp";
```

Comment:
This file is added, and there are two @client decorators, we are using this file as the entrance of code, there's no other generated code change in the loadTest case, which means we have ignored it.